### ISSUE TITLE
Illegal hardware instruction on MacOS

### ISSUE THREAD

**Issue by Raymo111** [CharaChorder/nexus#7]
```
python src/nexus/main.py -l DEBUG                                                                                                                   132 ↵
2023-07-29 16:57:27,820 - Listing words, limit -1, sort_by WordMetadataAttr.word, reverse False, case INSENSITIVE
2023-07-29 16:57:30,790 - Starting freqlogging
2023-07-29 16:57:30,797 - new_word_threshold=5, chord_char_threshold=30, allowed_keys_inchord={'P', 'C', 'Q', 'l', 'm', '3', 'Z', 'k', 'j', 'r', 'o', 's', 'q', '0', 'p', 'O', 'K', 'c', 'M', '8', '6', 'f', 'H', 'v', '4', 'i', 'Y', 'u', '2', 'd', 'F', '\', 'b', 'W', "'", '~', 'y', 'T', 'z', '1', '-', 'B', 'h', 'X', 'V', '5', 'g', 'a', 'w', '', 'e', 'G', 'x', '7', 'I', 'E', 'U', '/', 'n', 'D', 'J', 'L', 'R', 'S', '9', 't', 'N', 'A'}, modifier_keys={<Key.ctrl: <59>>, <Key.ctrl_r: <62>>, <Key.alt_r: <61>>, <Key.cmd_r: <54>>, <Key.alt: <58>>, <Key.cmd: <55>>}
[1]    24159 illegal hardware instruction
```

**Comment by Raymo111** [comment]
Upstream issue and may not be fixable, see
https://github.com/moses-palmer/pynput/issues/366
https://github.com/moses-palmer/pynput/issues/311
https://github.com/moses-palmer/pynput/issues/424

**Comment by Raymo111** [comment]
The bug is at this line:
https://github.com/CharaChorder/nexus/blob/862b66846d308851c3b7870df656c634f1363acc/src/nexus/Freqlog/Freqlog.py#L310
when called from:
https://github.com/CharaChorder/nexus/blob/862b66846d308851c3b7870df656c634f1363acc/src/nexus/GUI.py#L262

Interestingly, the next line (starting the mouse listener) runs fine (i.e. if moved to before the keyboard listener).

**Comment by Danc2050** [comment]
Attaching crash log:
[Problem Report for nexus-macos.txt](https://github.com/CharaChorder/nexus/files/13980998/Problem.Report.for.nexus-macos.txt)

**Comment by Raymo111** [comment]
Fixed in #134.
